services/cad-service/grpc_server.py

```
@@ -0,0 +1,93 @@
+    ExportFormat,
+    build_box,
+    export_solid_bytes,
+    get_export_descriptor,
+    serialize_box,
+)
+
+
+class CadService(pb2_grpc.CadServiceServicer):
+    """Implementation of the CAD gRPC service."""
+
+    async def GetBox(self, request: pb2.BoxRequest, context: grpc.aio.ServicerContext) -> pb2.BoxResponse:  # noqa: N802
```

Ruff (0.14.1)

Unused method argument: `context`

(ARG002)